spec.md

```diff
@@ -994,6 +995,7 @@ The following HTTP Headers are defined for this operation:
 | service_id* | string | MUST be the ID of a service from the catalog for this Service Broker. |
 | plan_id | string | If present, MUST be the ID of a plan from the service that has been requested. If this field is not present in the request message, then the Service Broker MUST NOT change the plan of the instance as a result of this request. |
 | parameters | object | Configuration parameters for the Service Instance. Service Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. See "Note" below. |
+| version | object | A free-form hash that represents the version to which the service instance should be updated to. Service Brokers SHOULD ensure that the client has provided a version supported by the broker. |
```

Comment:
I'm not sure I understand how this works. Each plan would now have a version associated with it, so why would they need to pass in a version on this operation? Isn't the plan ID good enough?  Did you mean for there to be some list of versions that a user can choose from?